pkg/client/scanner/scanner.go

```diff
@@ -68,7 +70,7 @@ func FromStr(name string) Tool {
 		}
 	}
 
-	return Default
+	return Tool{}
```

Comment:
Why the `FromStr` method  does not return a default scanner? 
What is the benefit?